alf/algorithms/sac_algorithm.py

```diff
@@ -214,7 +214,7 @@ def actor_loss_fn(dqda, action):
             if self._dqda_clipping:
                 dqda = tf.clip_by_value(dqda, -self._dqda_clipping,
                                         self._dqda_clipping)
-            loss = losses.element_wise_squared_loss(
+            loss = 0.5 * losses.element_wise_squared_loss(
```

Comment:
To  be consistent, we should also have 0.5 for ddpg_algorithm

Reply:
updated